backends/gdx-backend-lwjgl3/src/com/badlogic/gdx/backends/lwjgl3/Lwjgl3Application.java

```diff
@@ -562,6 +563,8 @@ static long createGlfwWindow (Lwjgl3ApplicationConfiguration config, long shared
 			try {
 				Class gles = Class.forName("org.lwjgl.opengles.GLES");
 				gles.getMethod("createCapabilities").invoke(gles);
+				// TODO: 06.10.23 Remove once https://github.com/LWJGL/lwjgl3/issues/931 is fixed
```

Comment:
This is very minor, but 06.10.23 is ambiguous and depends on culture-specific date formatting rules. I'm pretty sure you meant October 6, 2023, but my brain sees 06.10 and thinks June 10th. You could also just remove the date entirely, since the Git history should hold onto when this was entered.

Reply:
Agree. 2023-10-06 (ISO 8601) or 6 Oct 2023 would be clearer.

Reply:
IntelliJ just autogenerated the date, I don't think there is any point in keeping it. I removed it